app/modules/users/permissions/rules.py

```diff
@@ -387,9 +416,5 @@ def check(self):
 
 
 # Helpers to have one place that defines what users are privileged in all cases
-def user_is_privileged(user):
-    return user.is_staff or user.is_internal
-
-
 def owner_or_privileged(user, obj):
```

Comment:
Yes it makes sense to move the above logic to user but I think that means that this should move into user too